cloud_governance/cloud_resource_orchestration/aws/ec2/monitor_tickets.py

```diff
@@ -81,19 +85,29 @@ def __send_ticket_status_alerts(self, tickets: dict, ticket_status: str):
                                   'user_manager': '', 'project': description.get('Project'), 'owner': f'{description.get("FirstName")} {description.get("LastName")}'.upper(), 'total_spots': 0,
                                   'total_ondemand': 0, 'AllocatedBudget': [], 'instances_list': [], 'instance_types_list': []}
                         self.__es_operations.upload_to_elasticsearch(index=self.es_cro_index, data=source, id=ticket_id)
-                    if description.get('TicketOpenedDate').date() != datetime.now().date():
+                    current_date = datetime.now().date()
+                    ticket_opened_date = description.get('TicketOpenedDate').date()
+                    if ticket_opened_date != current_date:
                         user = description.get('EmailAddress').split('@')[0]
                         manager = description.get('ManagerApprovalAddress').split('@')[0]
                         cc = self.__default_admins
                         subject = body = to = None
+                        ticket_opened_days = (current_date - ticket_opened_date).days
                         if ticket_status == self.NEW:  # alert manager if didn't take any action
                             to = manager
-                            subject, body = self.__mail_message.cro_request_for_manager_approval(manager=to, request_user=user, cloud_name=self.__cloud_name, ticket_id=ticket_id, description=description)
-                            cc.append(description.get('EmailAddress'))
+                            extra_message = ''
+                            if self.__manager_escalation_days <= ticket_opened_days <= self.__manager_escalation_days + 2:
+                                manager_of_manager = self.__ldap_search.get_user_details(user_name=manager)
+                                to = manager_of_manager.get('ManagerId', '')
+                                extra_message = f"Your associate/Manager: [{manager}] doesn't approve this request.<br/>The user {user} is waiting for approval for last {ticket_opened_days} days.<br/>" \
+                                                f"Please review the below details and approve/ reject"
+                            elif ticket_opened_days >= self.__manager_escalation_days + 2:
+                                to = self.__global_admin_name
+                                extra_message = f"<b>CloudAdmin</b> -  Missing manager approval.<br />The user {user} is waiting for approval for last {ticket_opened_days} days.<br/>Please review the below details and approve/reject"
```

Comment:
if you sent it to cloud admin, why you need to mention:
`<b>CloudAdmin</b> -  Missing manager approval `
I think you should write something like that:

```
Hi <Cloud admin name>,
The user <user> is waiting for manager <manager> approval more than 5 days.
Please review below ticket details and approve/reject the ticket.
```